Dockerfile

```diff
@@ -1,9 +1,10 @@
-FROM registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.16-openshift-4.9 AS builder
+FROM registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.17-openshift-4.11 AS builder
```

Comment:
It should use 4.10 images.
Perhaps we should delete unused Dockerfiles (without .rhel7 suffix) eventually.

Reply:
oops sorry I just blindly cherry-picked. yeah we could delete these. currently `sync_bundle` script refers those images and that needs to be updated too.

Reply:
fixed